src/SqlView/SqlView.component.js

```diff
@@ -138,8 +138,9 @@ class SqlView extends Component {
 
     /* eslint-disable react/no-array-index-key */
     renderCell = (cell, index) => {
-        const isObject = typeof cell === 'object' && cell != null;
-        const formattedValue = isObject ? JSON.stringify(cell, null, 2) : cell;
+        const formattedValue =
+            cell === null ? '' :
+            (typeof cell === 'object' ? JSON.stringify(cell, null, 2) : String(cell));
```

Comment:
This is quite hard to parse. I think this should be outsourced into its own function for better readability:

```js
const formatCellValue = (cell) => {
  if (cell === null) {
    return null
  }

  if (typeof cell === 'object') {
    return JSON.stringify(cell, null, 2)
  }

  return String(cell)
}
```

Reply:
Logic moved to function

Reply:
Hi @Mohammer5 : I've moved to logic into a function. If you'd like any other changes, let me know